fem/nonlinearform_ext.cpp

```diff
@@ -40,20 +40,18 @@ double PANonlinearFormExtension::GetGridFunctionEnergy(const Vector &x) const
    elemR->Mult(x, xe);
    for (int i = 0; i < dnfi.Size(); i++)
    {
-      energy += dnfi[i]->GetGridFunctionEnergyPA(xe);
+      energy += dnfi[i]->GetStateEnergyPA(xe);
```

Comment:
`Energy` vs `State`: would `LocalEnergy` work?

Reply:
It's not "Energy" vs "State", it's "GridFunction" -> "State" because this input is not a `GridFunction`-size (i.e. L-) vector. Replacing `GetStateEnergyPA` with `GetLocalEnergyPA` will emphasize that the returned value is local (to the MPI-rank), so it may work. Another option is `GetLocalStateEnergyPA`.

Any other preferences/opinions?

Reply:
I like `GetLocalStateEnergyPA`.

Reply:
Renamed to `GetLocalStateEnergyPA`.